### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by polosanya — CHANGES_REQUESTED** [review]
Your DEMO link doesn't work.
image

Also please add these files to your commit to fix this problem
image
If you have any difficulties - feel free to ask for help in your fe_chat)

**Comment by savchuksavchuk** [comment]
sorry, I forgot  to add new files to index

File: src/components/CommentInfo/CommentInfo.tsx [hunk 1]
```diff
@@ -1,5 +1,23 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+export const CommentInfo: React.FC<Comment> = ({ name, email, body }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{name}</strong>
```
Comment (liliia-livandovskaya) [inline src/components/CommentInfo/CommentInfo.tsx:7]:
```suggestion
      <strong className="CommentInfo__name">
        {name}
      </strong>
```

File: src/components/PostList/PostList.tsx [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  posts: Post[],
+}
+
+export const PostList: React.FC<Props> = (props) => (
+  <div className="PostList">
+    {props.posts.map(post => (
+      <PostInfo {...post} />
```
Comment (liliia-livandovskaya) [inline src/components/PostList/PostList.tsx:12]: add <li> tag with `key`

Comment (liliia-livandovskaya) [inline src/components/PostList/PostList.tsx:12]:
```suggestion
      <PostInfo {post} />
```

File: src/components/CommentList/CommentList.tsx [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+      <>
+        <hr />
+
+        <b data-cy="NoCommentsMessage">No comments yet</b>
+      </>
+    );
+  }
+
+  return (
+    <div className="CommentList">
```
Comment (liliia-livandovskaya) [inline src/components/CommentList/CommentList.tsx:21]:
```suggestion
    <ul className="CommentList">
```

File: src/components/CommentList/CommentList.tsx [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+      {props.comments.map(comment => (
+        <CommentInfo {...comment} />
```
Comment (liliia-livandovskaya) [inline src/components/CommentList/CommentList.tsx:23]:
```suggestion
        <CommentInfo {comment} />
```
add there <li> tag with key

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
GJ! Pay attention to the comments

**Review by polosanya — APPROVED** [review]
Good job!

**Pull request closed without merging**